### DISCUSSION TITLE
❓ Visible snippets

### DISCUSSION THREAD

**Discussion by jjanes0918** [isontheline/pro.webssh.net#290] [General]
Is there a way to display more snippets. I currently have six but only five show up

**Comment by isontheline** [comment]
You're right there are a max of 5 snippets fixed in the code (both iPhone and iPad) :
![image](https://user-images.githubusercontent.com/44212923/117874855-d05a7500-b2a1-11eb-9944-e74301ac3e34.png)

I think on iPad this maximum should be bigger.

Are you using an iPad?

**Comment by jjanes0918** [comment]
Nope iPhone 11pro max anyway that could be adjusted
Accessing them is probably the issue though

**Reply by isontheline** [reply]
Yes accessing them all is the issue :/
Do you need to access to them all for all your connections or only some as this https://github.com/isontheline/pro.webssh.net/issues/272 issue?

**Comment by jjanes0918** [comment]
Don’t worry too much about it I am using it more as a fun tool not for
business.

I do appreciate your emails back to me but please do not spend a lot of
time on this it is a wish list only for a project I am having fun with.

I have a project going on with 3-D printers that is allowing me to build a
Stargate just like from the movie and the TV series and we are able to dial
others stargate from our own to actually people across the world one of the
functionalities is to SSH into our gate and simply dial others as tests.
This is what I was doing instead of having to remember all of the addresses
I was using macros to dial them for me.  We have many users and I was using
one macro per user and in addition certain functional utilities I could
perform on my gate.

I did make the 2.99 payment to you to say thanks for building this app it
has the best functionality I could find with regard to macros and multiple
windows.

Thanks
Jeff

**Reply by isontheline** [reply]
@jjanes0918 Thank you a lot Jeff for your feedback, support and your purchase!

Have a nice day ☀️

**Comment by isontheline** [comment]
Hello @jjanes0918 👋

Does the issue https://github.com/isontheline/pro.webssh.net/issues/272 could help you?

**Reply by isontheline** [reply]
@jjanes0918 Any update about this?
